src/jit/lsraarmarch.cpp

```diff
@@ -771,6 +771,11 @@ void Lowering::TreeNodeInfoInitPutArgSplit(GenTreePutArgSplit* argNode, TreeNode
     }
     argNode->gtLsraInfo.setDstCandidates(m_lsra, argMask);
 
+    if (putArgChild->OperGet() == GT_FIELD_LIST)
+    {
+        NYI_ARM("LSRA: Oper for split struct argument is GT_FIELD_LIST");
```

Comment:
If I understand correctly, this is just because you haven't gotten to this step yet?